app/models/hackathon/digest/listings/by_location.rb

```diff
@@ -0,0 +1,33 @@
+module Hackathon::Digest::Listings::ByLocation
```

Comment:
What if called this module `Search` or `Nearby`

Reply:
`Search` doesn't really make sense, at least in my brain. "Lemme tell you about `Digest`s! They list `Hackathon`s, right now `ByLocation`, but maybe through other criteria later on!" Concerns like these allow explaining a complex system through structured writing. If you think `Nearby` makes more sense, that's fine with me, but it might be a bit misleading cause users can have multiple subscriptions to different locations.

Reply:
Notes from one-on-one:
- `ByLocation` is good. let's keep it!